joint_angles:
    type: "joint_angles"
    data:
        vertical_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
        #intermediate_pick_pose: [179.0, 17.0, 182.0, 231.0, 0.0, 56.0, 92.0]

        #values are in degrees
        perceive_table:
            [
                -11.865325927734354,
                21.10895347595215,
                129.39122009277344,
                -52.20489501953125,
                20.05187225341797,
                -110.82797241210935,
                38.08707809448242,
            ]
